plugin-flex-ts-template-v2/src/feature-library/chat-transfer/README.md

```diff
@@ -1,41 +1,74 @@
 
-It makes use of the existing worker directory but note that it removes the warm/consult transfer functionality as this is out of scope for this plugin.
+**Config options allows for two different options:**
 
-![alt text](screenshots/chat-transfer.gif)
+- _cold_transfer:_ Enables a ‘transfer’ icon for the task header that can be used to implement cold/blind transfer
+- _multi_participant:_ Enables a participants tab that is used to invite other agents to the conversation. This participants tab allows for adding and removing of agents so that multiple agents can be in the chat at the same time. This allows an agent to delay leaving the chat until another agent has joined.
 
-## Setup
+The two different features can be enabled/disabled independently. In the case of both being enabled and there is an invite that has been sent to an agent or queue then the cold_transfer option is disabled until an agent joins or the invite is canceled.
 
-A new task is created for each transfer in a configurable TaskRouter workflow. For correct routing the workflow should implement logic to route the task to a specific worker or to a queue based on queue name.
+### Cold Transfer
+![cold transfer](screenshots/chat-transfer.gif)
 
-The TaskRouter workflow sid (WWxxx) should be added to the .env file in the serverless directory before deploying the service to Twilio.
+### Multi participant chat
+![multiple partcipants](screenshots/multi-participant.gif)
+
+## Setup
 
+### Config
+As described above there are option flags for cold transfer and multi-participation.
+The features configuration options are:
+```javascript
+{
+ enabled: boolean;
+ cold_transfer: boolean;
+ multi_participant: boolean;
+}
+```
+### Serverless
+The TaskRouter workflow sid (WWxxx) should be added to the .env file in the serverless directory before deploying the service to Twilio.
 ```
 # CHAT TRANSFER
 TWILIO_FLEX_CHAT_TRANSFER_WORKFLOW_SID=WWxxx
 ```
 
-Task attributes have the following attributes added to the transferring Task:
+### TaskRouter Workflow
+
+A new task is created for each invite to an agent or queue. A TaskRouter workflow is used to route to the correct agent (using [Known Agent Routing](https://www.twilio.com/docs/taskrouter/workflow-configuration/known-agent-routing)) and requires a target for each queue that agents will invite agents from.
+As well as indicating the transfer target in task attributes it also adds the worker sids for agents that are currently in the chat. The workflow can use this task attribute to ensure that agents already in the chat are not considered for routing for transfers to a queue.
 
+The TaskAttributes that are set by the plugin are:
 ```
 transferTargetType - set to either worker or queue
 transferTargetSid - will be set to the worker sid in the case of target type == worker
 transferQueueName - TaskRouter friendly name for the queue in the case of target type == queue
+workerSidsInConversation - string array of workers in the conversation
```

Comment:
It looks like this property name is now `workersToIgnore` instead of `workerSidsInConversation`. If that's correct, we just need to update the README and sample workflow config to match.

Reply:
Nevermind, I missed where `workerSidsInConversation` is actually a property of `workersToIgnore` object, then `workersToIgnore` is spread out in the attributes, so the documentation and example is correct. I apologize for my confusion. Please disregard this comment.